src/compiler/extensions/debugger.js

```diff
@@ -19,11 +19,11 @@ let debuggerExtension = {
   instructions: {
     TORNADO_DEBUGGER: {
       enter(node, ctx, frameStack) {
-        let inner = frameStack.current(),
-            outer = inner;
+        let child = frameStack.current(),
+            parent = child;
         return {
           type: 'insert',
-          options: {key: node[1].key, frameStack: [inner, outer], item: node.stackItem, ctx}
+          options: {key: node[1].key, frameStack: [child, parent], item: node.stackItem, ctx}
```

Comment:
interesting.  you don't need it because it's not used by the codeGen step below it.

But I see your point. We need extensions to use a stable API. So we could reintroduce the Context  class, that would wrap wrap these.